spec/firebase_app_distribution_api_client_spec.rb

```diff
@@ -3,7 +3,12 @@
   let(:fake_binary_contents) { "Hello World" }
   let(:fake_binary) { double("Binary") }
   let(:headers) { { 'Authorization' => 'Bearer auth_token' } }
-
+  let(:upload_headers) do
```

Comment:
If `upload_headers` is only used inside the `describe '#upload_binary'` block, then you can move this let down there. In general I think it's a good idea to scope these variables as low as possible just to avoid having declaration overload up here at the top of the file.

Reply:
That makes sense thank you!